### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by efirs**
* Fix transaction rollback on failure
* Automatically retry implicit transactions
* Update client to newest and add high level client integration tests
* Removing logging from -v output
* Fix FDB connection when test on the hosts and server in the docker
* Various clean ups

### CODE REVIEW

**Review by himank — APPROVED** [review]

File: server/metadata/encoding/schema_test.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 )
 
 func TestSchemaSubspace(t *testing.T) {
+	t.Skip("aaa")
```
Comment (himank) [inline server/metadata/encoding/schema_test.go:31]: remove it

File: store/kv/fdb.go [hunk 2]
```diff
@@ -481,15 +510,16 @@ func (t *ftx) Commit(ctx context.Context) error {
...
+
+			if err != nil && ep.Code == 1020 {
 				err = ErrConflictingTransaction
-			} else if err1 := t.tx.OnError(ep).Get(); err1 != nil {
-				err = err1
 			}
 		}
 
 		if err != nil {
+			t.tx.Cancel()
```
Comment (himank) [inline store/kv/fdb.go:522]: do we need this cancel in fdbkv as well?

Comment (efirs) [reply store/kv/fdb.go:522]: Yes, please check `txWithRetry` it has `defer tr.Cancel()`

**Review by himank — DISMISSED** [review]

File: store/kv/fdb.go (hunk 2 above)
Comment (himank) [reply store/kv/fdb.go:522]: ah, right.

**Review by himank — APPROVED** [review]

**Pull request merged**